src/sqlancer/common/ast/SelectBase.java

```diff
@@ -50,6 +50,10 @@ public void setGroupByExpressions(List<T> groupByExpressions) {
         this.groupByExpressions = groupByExpressions;
     }
 
+    public void clearGroupByExpressions() {
```

Comment:
I think we don't need a separate method here and below. We can just use `null` as an argument to clear the expression(s). For the `GROUP BY` expressions, we can check whether the argument is `null` and if so, set an empty list.

Reply:
Directly setting `null` will leave some issues, such as `null pointer`. Here, I need to make it an empty `collection`, instead of `null.

Reply:
Right. Above, I meant, that in the setter, we can just check whether the argument is `null`, and if so, set an empty list.

Reply:
https://github.com/sqlancer/sqlancer/blob/main/src/sqlancer/common/ast/SelectBase.java#L48 If the argument is `null`, it would throw an assertion. I guess it may break other code that depends on this function.

Reply:
Alright, then let's use the new `clear` methods that you introduced rather than changing the behavior of the `SelectBase`. I think it would unlikely, though, that existing code relies on an exception being thrown here.